### PR TITLE
WD-8308 - feat: Add comments to errors that should continue to only be logged to the console

### PR DESCRIPTION
**Pull request by vladimir-cucu**
## Done

- Add comments to errors that should continue to only be logged to the console

## Details

- https://warthogs.atlassian.net/browse/WD-8308

### CODE REVIEW

File: src/serviceWorker.js [hunk 1]
```diff
@@ -94,6 +94,8 @@ function registerValidSW(swUrl, config) {
       };
     })
     .catch((error) => {
+      // Entire code is autogenerated from CRA. The dashboard will continue to
+      // work if we get an error here. Logged for debugging purposses.
```
Comment (huwshimi) [inline src/serviceWorker.js:98]: I don't think we should modify this file as it is autogenerated.

File: src/setupTests.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ const animateMock = () => {
 if (!window.HTMLDivElement.prototype.animate) {
   window.HTMLDivElement.prototype.animate = animateMock;
 } else {
+  // Error is related to tests. Not shown in UI. Logged for debugging purposses.
```
Comment (huwshimi) [inline src/setupTests.js:15]: I don't think we need this comment as it's not part of the shipped client.

File: src/store/store.ts [hunk 3]
```diff
@@ -15,6 +15,8 @@ if (process.env.NODE_ENV !== "production" && process.env.REACT_APP_MOCK_STORE) {
   try {
     preloadedState = JSON.parse(process.env.REACT_APP_MOCK_STORE);
   } catch (error) {
+    // Error is related to tests and is not accessible in production.
+    // Not shown in UI. Logged for debugging purposses.
```
Comment (huwshimi) [inline src/store/store.ts:19]: I don't think we need this comment as it's excluded from production, but fine to leave it here I guess.

File: src/hooks/useLocalStorage.ts [hunk 4]
```diff
@@ -11,6 +11,7 @@ function useLocalStorage<V>(
       const item = window.localStorage.getItem(key);
       return item ? JSON.parse(item) : initialValue;
     } catch (error) {
+      // Not shown in UI. Logged for debugging purposses.
```
Comment (huwshimi) [inline src/hooks/useLocalStorage.ts:14]: A spelling fix here for all these comments.
```suggestion
      // Not shown in UI. Logged for debugging purposes.
```

**Review by huwshimi — APPROVED** [review]
A few small comments.

File: src/hooks/useLocalStorage.ts (hunk 4 above)
Comment (vladimir-cucu) [reply src/hooks/useLocalStorage.ts:14]: I've fixed this and also fixed the same issue on line 28. Thanks for the review!

File: src/store/store.ts (hunk 3 above)
Comment (vladimir-cucu) [reply src/store/store.ts:19]: I've removed it.

**Pull request merged**